The Hindu (old).js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "9499c586-d672-42d6-9ec4-ee9594dcc571",
 	"label": "The Hindu (old)",
 	"creator": "Prashant Iyengar and Michael Berkowitz",
-	"target": "^https?://(www\\.)?hindu\\.com/",
+	"target": "^https?://(www\\.)?thehindu\\.com/lr/",
```

Comment:
let's also add `item.ISSN = "0971-751X";` and remove `Zotero.wait()` and `Zotero.done()`